_studio/shared/src/cm_mem_copy.cpp

```diff
@@ -2616,23 +2616,23 @@ mfxStatus CmCopyWrapper::InitializeSwapKernels(eMFXHWType hwtype)
 #if !(defined(AS_VPP_PLUGIN) || defined(UNIFIED_PLUGIN) || defined(AS_H264LA_PLUGIN))
```

Comment:
you are kidding me... and here we exclude all the kernels.

Reply:
None of them defined in library build. So exactly opposite is true. For now only AS_H264LA_PLUGIN part remained relevant.

Reply:
I am trying to say: why these ifdefs are not aligned with the same ifdefs to include header files?